test/unit/serialize.test.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @jest-environment @edge-runtime/jest-environment
```

Comment:
Should we test this on the jsdom environment as well as the edge environment?
(I'm not sure the difference between those environments, this might not be necessary)

Reply:
From my perspective, `edge-runtime` is like a strict subset of `nodejs`, so if it works for `edge-runtime`, it should also work for `nodejs`.

I feel like it might not be necessary for now.